homework-2/scss/style.scss

```diff
@@ -0,0 +1,1385 @@
+}
+.link-map{
+  font-family: "Aclonica-Regular", sans-serif;
+  font-size: 12px;
+  line-height: 20px;
+  color: #22bf95;
+  transition: 1s;
+  &:hover{
+    color: #7f7f7f;
+    text-decoration: underline;
+  }
+}
+.social{
+  padding-bottom: 24px;
+  &-inner{
+    border-top: 2px solid #b2b2b2;
+  }
+  &-list{
+    display: flex;
+    align-items: center;
+    justify-content: space-around;
+    margin: 29px 14px;
+  }
+}
+svg .social-hover{
+  transition: 200ms;
+  transition-timing-function: ease-in-out;
+  -webkit-transition: 200ms;
+  -webkit-transition-timing-function: ease-in-out;
+}
+svg:hover .social-hover{
+  fill: #22bf95;
+}
+.footer-logo-img{
+  display: block;
+  margin: 0 auto;
+}
+@include tablet{
+  .container{
+    max-width: 767px;
+    padding: 0 25px;
+    margin: 0 auto;
+  }
+  .hero{   
+    &-title{
+      max-width: 510px;
+      padding: 120px 0 0;     
+    }
+    &-text{
+      max-width: 510px;
+      &::before{
+        height: 90px;
+      }
+    }  
+  }
+  .about{
+    &-items{
+      flex-direction: row;
+      align-items: end;
+    }
+    .about-img{
+      max-width: 350px;
+    }
+    &-item-right{
+      order: 2;
+      margin-top: 90px;
+      margin-left: 61px;
+    }
+    &-item-left{
+      order: 1;
+    }
+  }
+  .essensials{
+    &-title{
+      max-width: 785px;
+    }
+    &-cards{
+      margin-top: 46px;
+    }
+  }
+  .essensials-card{
+    display: flex;
+    justify-content: center;
+  }
+  .card{
+    margin-top: 30px;
+    &-second{
+      margin-left: 30px;
+    }
+    &-btn{
+      max-width: 230px;
+    }
+  }
+  .slider{
+    .slide-title{
+      max-width: 451px;
+      margin: 20px auto 0;
+      font-size: 18px;
+    }
+    &-text{
+      max-width: 451px;
+      margin: 20px auto 0;
+      font-size: 18px;
+    }
+  }
+  .slick-slide img{
+    max-width: 470px;
+    width: 100%;
+    margin: 20px auto 0;
+  }
+  .services-item-text{
+    padding: 0 100px;
+  }
+  .planse-card{
+    padding: 35px 5px 25px;
+  }
+  .card-item-wrapper::before{
+    width: 140px;
+  }
+  .card-item{
+    font-size: 10px;
+  }
+  .description{
+    max-width: 700px;
+  }
+}
+@include desktop{
+  .container{
+    max-width: 1200px;
+    width: 100%;
+    padding: 0 84px;
+  }
+  .btn{
+    max-width: 200px;
+    width: 100%;
+    font-size: 12px;
+  }
+  .title{
+    font-size: 42px;
+    line-height: 54px;
+  }
+  .text{
+    font-size: 13px;
+    line-height: 24px;
+  }
+  .hamburger-menu{
+    display: none;
+  }
+  .header{   
+  .header-button{
+      flex-direction: row;
+      max-width: 290px;
+      margin: 0 0 0 30px;
+      top: 0;
+      & .btn{
+        color: #fff;
+        border: 1px solid #fff;
+      }
+      & .get-btn{
+       margin-right: 20px;     
+      }
+      & .log-btn{
+        margin-top: 0;
+      }
+    }
+  }
+  .header-menu{
+    flex: 1;
+  }
+  .menu-body{
+    position: static;
+    transform: translateY(0px);
+    background: transparent;   
+    .menu-list{
```

Comment:
`.menu-list` class used in header and in footer. But they are completely different. Better use another class names. It could be `header-menu` and `footer-menu`. Or `header-nav` and `footer-nav`

Reply:
done